nautobot/extras/tests/test_relationships.py

```diff
@@ -14,24 +13,24 @@
 )
 from nautobot.core.tables import RelationshipColumn
 from nautobot.core.testing import TestCase
+from nautobot.core.testing.models import ModelTestCases
 from nautobot.core.utils.lookup import get_route_for_model
 from nautobot.dcim.models import Device, Platform, Rack, Location, LocationType
 from nautobot.dcim.tables import LocationTable
 from nautobot.dcim.tests.test_views import create_test_device
-from nautobot.ipam.factory import VLANGroupFactory
 from nautobot.ipam.models import VLAN, VLANGroup
 from nautobot.extras.choices import RelationshipRequiredSideChoices, RelationshipSideChoices, RelationshipTypeChoices
 from nautobot.extras.models import Relationship, RelationshipAssociation, Status
 
 
-class RelationshipBaseTest(TestCase):
+class RelationshipBaseTest:
     @classmethod
     def setUpTestData(cls):
         cls.location_ct = ContentType.objects.get_for_model(Location)
         cls.rack_ct = ContentType.objects.get_for_model(Rack)
         cls.vlan_ct = ContentType.objects.get_for_model(VLAN)
 
-        cls.locations = Location.objects.all()[:5]
+        cls.locations = Location.objects.get_for_model(Rack).get_for_model(VLAN)[:5]
```

Comment:
Should we assert something like `len(cls.locations) == 5` or something after this line?

Reply:
Yeah, we should. Since this is a `classmethod` we can't do `self.assertEqual()` but I can maybe just add a plain `assert`?